src/player.h

```diff
@@ -753,9 +761,9 @@ class Player final : public Creature, public Cylinder
 	{
 		if (!sendAll) {
 			// update one slot
-			Thing* slotThing = getThing(CONST_SLOT_RIGHT);
+			auto slotThing = getThing(CONST_SLOT_RIGHT);
 			if (slotThing) {
-				Item* slotItem = slotThing->getItem();
+				auto slotItem = slotThing->getItem();
```

Comment:
Could this be `const auto`
```suggestion
				cont auto slotItem = slotThing->getItem();
```